pvlib/singlediode.py

```diff
@@ -684,7 +675,8 @@ def _pwr_optfcn(df, loc):
     Function to find power from ``i_from_v``.
     '''
 
-    I = _lambertw_i_from_v(df['r_sh'], df['r_s'],           # noqa: E741, N806
-                           df['nNsVth'], df[loc], df['i_0'], df['i_l'])
+    I = _lambertw_i_from_v(df[loc], df['photocurrent'],
```

Comment:
Feel free to rename this internal `I` here and elsewhere to mollify stickler. Not important.

Reply:
Ok, I renamed `I` to `current`. To make the rest of the code match, I changed `I` to `current` and `V` to `voltage` everywhere.

Reply:
Never mind, I changed it only at the stickler comments. `I` is also used as part of a variable name in many places.